### PR TITLE
Update what is kubernetes copy

### LINKED ISSUE #8753: Copy update - /kubernertes/what-is-kubernetes
Per the [copy doc](https://docs.google.com/document/d/19ZAsyujRHjOBIKP15F-4Z9L24KpfoYrx0YpL-0gO6sE/edit?ts=5fb3addf#)

### PR DESCRIPTION
**Pull request by ClementChaumel**
## Done

- update the copy on what is kubernetes as per the copydoc

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/kubernetes/what-is-kubernetes
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes #8753

### CODE REVIEW

**Review by bethcollins92 — CHANGES_REQUESTED** [review]
Looking good! Just in the logo cloud you're missing Amazon, Microsoft and Rancher Labs. You should be able to find them here https://manager.assets.ubuntu.com/

image

**Comment by anasereijo** [comment]
hey @ClementChaumel just one tiny thing:
can you please make the background of K8's resources (bottom of the page) white?

thanks!

**Comment by anasereijo** [comment]
@ClementChaumel perfect, thanks!

File: templates/kubernetes/what-is-kubernetes.html [hunk 1]
```diff
@@ -644,26 +642,13 @@ <h4 class="p-heading-icon__title">Whitepaper</h4>
         </div>
         <h3 class="p-heading--four">
           <a class="p-link--external" href="https://pages.ubuntu.com/container-whitepaper.html">
```
Comment (bethcollins92) [inline templates/kubernetes/what-is-kubernetes.html:644]: Sorry I missed this, this link should be: 

```suggestion
          <a href="/engage/kubernetes-deployment-enterprise-whitepaper">
```

**Review by bethcollins92 — APPROVED** [review]
LGTM 👍 Just one small change in the link

**Pull request merged**